code/addons/onboarding/src/App.tsx

```diff
@@ -27,7 +27,7 @@ export default function App({ api }: { api: API }) {
   const skipOnboarding = useCallback(() => {
     // remove onboarding query parameter from current url
     const url = new URL(window.location.href);
-    const path = decodeURIComponent(url.searchParams.get('path'));
+    const path = decodeURIComponent(url.searchParams.get('path') as any);
```

Comment:
The commit `b2caeba335b61f76ea27b51680943f60b286054b` has added a lot of TypeScript "fixes" where actually files where adjusted to fit the new TypeScript version and configuration. This PR only aims to migrate the onboarding from one repo into another and therefore fixing these TypeScript errors is indeed not part of the work. Instead of annotating with `any` or using the explanation mark operator to suppress `undefined`, my proposal would be to just leave everywhere a `ts-expect-error` comment. Then, if we think we would like to improve typings and even catch and handle undefined behavior programmatically, we can search through `ts-expect-error` comments and take care of these uncovered scenarios. WDYT?